src/settings_type.h

```diff
@@ -313,9 +313,12 @@ struct NetworkSettings {
 	std::string server_invite_code_secret;                ///< Secret to proof we got this invite code from the Game Coordinator.
 	std::string server_name;                              ///< name of the server
 	std::string server_password;                          ///< password for joining this server
+	std::vector<std::string> server_authorized_keys; ///< public keys of clients that are authorized to connect to the game
```

Comment:
nitpick: please start sentences with capital and end with dot. I know I know, most don't have that. But new entries we have been changing :)